client/src/pages/sponsors.js

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable max-len */
+import React, { Fragment } from 'react';
+import Helmet from 'react-helmet';
+import { Grid, Col, Row } from '@freecodecamp/react-bootstrap';
+
+import { Link, Spacer } from '../components/helpers';
+
+import './sponsors.css';
+
+const SponsorsPage = () => {
+  return (
+    <Fragment>
+      <Helmet title='Sponsors who help freeCodeCamp through financial and in-kind sponsorship | freeCodeCamp.org' />
```

Comment:
We can still enforce the `max-len` if you rearrange the props to use string concatenation, like so:

```jsx
<Helmet
  title={
    'Sponsors who help freeCodeCamp through financial and in-kind ' +
    'sponsorship | freeCodeCamp.org'
  }
/>
```